content/doc/book/installing/index.adoc

```diff
@@ -613,6 +613,35 @@ ZFS integration with Jenkins should be considered.
 
 include::doc/book/installing/_setup-wizard.adoc[]
 
+=== Offline Jenkins Installation
+
+This section describes how to install Jenkins on a machine
+that does not have an internet connection.
+
+To install Jenkins itself, download the appropriate war file and
+transfer it to your machine.
+
+Plugins are a different matter, due to dependency requirements. Two
+obvious choices are: +
+1) Install Jenkins and add required plugins on a network-attached
+computer. Make an archive of the Jenkins directory structure and
+transfer it to the offline machine. +
+2) Transfer Jenkins and plugin files separately and install on offline
+computer.
+
+If you want to transfer the individual plugins, you'll need to retrieve
+all dependencies as well. There are several dependency retrieval scripts
```

Comment:
I suggest to replace the scripts in this section by https://github.com/jenkinsci/plugin-installation-manager-tool

Reply:
Agreed.  I wonder if @stopalopa would be willing to help in this week's Hackfest efforts?  She might have really good recommendations to improve this section

Reply:
@oleg-nenashev 
So here the step to install the Jenkins will remain same.
But for plugin installation I can provide a link to https://github.com/jenkinsci/plugin-installation-manager-tool

Is that what you meant?

Reply:
@oleg-nenashev 
Updated please have a look